src/pluginHost.node/api/methods/openUrl.ts

```diff
@@ -0,0 +1,34 @@
+import { EnvironmentPlatform, openUrlFn } from 'einstein'
+
+import { spawn } from './spawn'
+
+export const openUrl: openUrlFn = async (platform: EnvironmentPlatform, url: string) => {
```

Comment:
suggestion: platform can be fetched from `Environment` (`src/pluginHost.node/Environment.ts`) directly

Reply:
fixed.